ChatQnA/docker/ui/react/nginx.conf

```diff
@@ -11,10 +11,17 @@ server {
   location / {
     root /usr/share/nginx/html;
     index index.html index.htm;
-    try_files $uri $uri/ /index.html =404;
 
     location ~* \.(gif|jpe?g|png|webp|ico|svg|css|js|mp4|woff2)$ {
       expires 1d;
     }
   }
+
+  location "/v1/chatqna" {
+      proxy_pass $CHAT_BASE_URL;
```

Comment:
is the environment variable correct ?  compose.yaml is setting `APP_BACKEND_SERVICE_ENDPOINT` and `APP_DATA_PREP_SERVICE_URL` as env variables according to `https://github.com/opea-project/GenAIExamples/blob/main/ChatQnA/docker/xeon/README.md?plain=1#L403`.

Reply:
@jaswanth8888 thanks for pointing this out. 

In the other UI the same variable is given the [name](https://github.com/opea-project/GenAIExamples/blob/08f57fa54a3cba01ad80c76fc94e5cbed2b86c7d/ChatQnA/docker/xeon/compose.yaml#L174) `CHAT_BASE_URL`.
I think I would change this README (and the one for Gaudi and Gpu) to use the same env var name. 
* `APP_BACKEND_SERVICE_ENDPOINT` -> `CHAT_BASE_URL`

I don't see any point in having 2 names for the same thing. 
I would do the same thing for 
* `APP_DATA_PREP_SERVICE_URL` -> `UPLOAD_FILE_BASE_URL`